tensorflow_serving/tools/docker/Dockerfile.devel

```diff
@@ -28,7 +28,7 @@ RUN curl -fSsL -O https://bootstrap.pypa.io/get-pip.py && \
 
 # Set up grpc
 
-RUN pip install enum34 futures six && \
+RUN pip install enum34 futures mock six && \
     pip install --pre protobuf>=3.0.0a3 && \
```

Comment:
While we are here, would it be possible to add `'protobuf>=3.0.0a3'`, it seems without it will touch a file on the root filesystem named `=3.0.0a3`, and I am not sure what version of protobuf is truly installed due to that.